packages/message-manager/src/AbstractMessageManager.ts

```diff
@@ -33,6 +33,8 @@ export interface AbstractMessage {
   type: string;
   rawSig?: string;
   securityProviderResponse?: Map<string, Json>;
+  rawData?: string;
+  error?: string;
```

Comment:
I tried adding an additional field for the encryption key in my PR but it didn't work out, so I ended up using `rawSig`.

The reason is that the `SignController` and `EncryptionPublicKeyController` `StateMessage` type uses `Required<AbstractMessage>`, so this would force them to populate `rawData` and `error`.

The bigger problem here is that you also need a field for the error, which was not the case in my PR.

Reply:
I was going to suggest the same, it's only used internally and what matters is what is resolved back to the promise, so we can reuse `rawSig` here rather than have an alternate property.

We can make a future ticket to rename this to `result` maybe to be more generic.

Reply:
Thanks guys, I just fixed these regarding your comments.